### DISCUSSION TITLE
Would Monai be open to contributions to integrate the reading of biosignal data?

### DISCUSSION THREAD

**Discussion by bruAristimunha** [Project-MONAI/MONAI#6991] [Ideas]
First of all, I want to say a huge thanks for the awesome library you're maintaining. It's been a real lifesaver for my research, and the time I spent with the generative team at KCL really helped me grasp how it all works. So, big thanks!

Now, here's the thing I noticed. Your library has support for bio-signals, which is great (https://docs.monai.io/en/stable/_gen/monai.transforms.signal.array.html#module-monai.transforms.signal.array). But it seems like there's a gap when it comes to reading bio-signals.

I've got a suggestion, and I'm even up for doing the work on it. How about we add a bio-signal reader that uses a versatile library like mne-python to handle all sorts of bio-signal formats automatically? Think of it as something similar to PilReader, but for bio-signals. This would make life so much easier for researchers diving into monai with bio-signals.

Would Monai be open to having this new reader as part of the package?

**Comment by KumoLiu** [comment]
Hi @bruAristimunha, sure, welcome to your contribution.
You can refer to this [contribution guide](https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md) for how to create a PR. 
Thanks!
